### PR TITLE
feat(Query): new navigation tab [YTFRONT-4235]

### PR DESCRIPTION
**Pull request by SimbiozizV**
https://nda.ya.ru/t/Jg9hF0-976kNhZ

### CODE REVIEW

File: packages/ui/src/ui/pages/query-tracker/Navigation/api/loadSyncTableRequest.ts [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+type LoadStaticTableRows = (props: {
+    path: string;
+    cluster: string;
+    schema: NavigationTableSchema[];
+    limit: number;
+    output_format: Record<string, any> | string;
+}) => Promise<{columns: string[]; rows: UnipikaValue[]; yqlTypes: TypeArray[] | null}>;
+
+export const loadSyncTableRequest: LoadStaticTableRows = async ({
```
Comment (ma-efremoff) [inline packages/ui/src/ui/pages/query-tracker/Navigation/api/loadSyncTableRequest.ts:23]: `Sync` should be replaced with `Static`
What is the reason to have `Request` suffix in the name?
Is it possible to have reusable function with/from Navigation?

File: packages/ui/src/ui/store/selectors/navigation/content/map-node.js [hunk 2]
```diff
@@ -133,7 +133,7 @@ export const getSelected = (state) => state.navigation.content.mapNode.selected;
 export const getLastSelected = (state) => state.navigation.content.mapNode.lastSelected;
 const getTextFilter = (state) => state.navigation.content.mapNode.filter;
 
-const getSortState = (state) => state.tables[NAVIGATION_MAP_NODE_TABLE_ID];
+export const getSortState = (state) => state.tables[NAVIGATION_MAP_NODE_TABLE_ID];
```
[the remark this answers is not in the document]
Comment (SimbiozizV) [reply packages/ui/src/ui/store/selectors/navigation/content/map-node.js:136]: This selector allows us to get field and asc. There's nothing else there. I use it to get data in the composite selector selectNodeListByFilter via createSelector. 

I don't think that the 2 fields obtained from this selector will change often. Therefore, the threat of redrawing is minimal here

File: packages/ui/src/ui/pages/query-tracker/Navigation/api/loadSyncTableRequest.ts (hunk 1 above)
Comment (SimbiozizV) [reply packages/ui/src/ui/pages/query-tracker/Navigation/api/loadSyncTableRequest.ts]: I added the suffix to indicate that this function is not an action, but rather executes a request and processes the result.

The fact is that both implementations differ in the parameters of the ytApiV3Id.readTable function. For side navigation, it is somewhat simpler

File: packages/ui/src/ui/pages/query-tracker/module/queryNavigation/actions.ts [hunk 3]
```diff
@@ -0,0 +1,293 @@
...
+import {insertTextWhereCursor} from '../../Navigation/helpers/insertTextWhereCursor';
+import {getCellSize, getPageSize} from '../../../../store/selectors/navigation/content/table-ts';
+import {editor as monacoEditor} from 'monaco-editor';
+import {getQueryDraft} from '../query/selectors';
+import {getRequestOutputFormat} from '../../../../utils/navigation/content/table/table';
+import {getDefaultTableColumnLimit} from '../../../../store/selectors/settings';
+import {isYqlTypesEnabled} from '../../../../store/selectors/navigation/content/table';
+import {getCurrentUserName} from '../../../../store/selectors/global';
+import {getTableYsonSettings} from '../../../../store/selectors/thor/unipika';
+import {prepareColumns} from '../../../navigation/content/Table/DataTableWrapper/DataTableWrapper';
```
[the remark this answers is not in the document]
Comment (SimbiozizV) [reply packages/ui/src/ui/pages/query-tracker/module/queryNavigation/actions.ts:49]: But this is a component, not a helper. Just with an incorrect name.

File: packages/ui/src/ui/store/selectors/navigation/content/map-node.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (SimbiozizV) [reply packages/ui/src/ui/store/selectors/navigation/content/map-node.js]: I corrected it

**Review by ma-efremoff — APPROVED** [review]

**Pull request merged**